odoo/addons/base/models/ir_cron.py

```diff
@@ -279,72 +292,100 @@ def _acquire_one_job(cls, cr, job_ids):
         return cr.dictfetchone()
 
     @classmethod
-    def _process_job(cls, db, cron_cr, job):
-        """ Execute a cron job and re-schedule a call for later. """
-
-        # Compute how many calls were missed and at what time we should
-        # recall the cron next. In the example bellow, we fake a cron
-        # with an interval of 30 (starting at 0) that was last executed
-        # at 15 and that is executed again at 135.
-        #
-        #    0          60          120         180
-        #  --|-----|-----|-----|-----|-----|-----|----> time
-        #    1     2*    *     *     *  3  4
-        #
-        # 1: lastcall, the last time the cron was executed
-        # 2: past_nextcall, the cron nextcall as seen from lastcall
-        # *: missed_call, a total of 4 calls are missing
-        # 3: now
-        # 4: future_nextcall, the cron nextcall as seen from now
-
+    def _execute_job(cls, job):
```

Comment:
Please move this method below `_process_job()`, as the file is organized top-down:
* `_process_jobs()` process all jobs by calling `_acquire_one_job()` and `_process_job()`
* `_process_job()` processes a particular job by calling `_execute_job()`
* `_execute_job()` actually runs the particular job in a loop by calling `_callback()`
* `_callback()` runs the action associated to the job